Tests/Pcap++Test/Tests/IpMacTests.cpp

```diff
@@ -503,53 +537,87 @@ PTF_TEST_CASE(TestIPv4Network)
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "ffff:e000::",                             19,  "39e1:e000::",                             "39e1:e000::1",           "39e1:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      },
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "ffff:8000::",                             17,  "39e1:8000::",                             "39e1:8000::1",           "39e1:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      },
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "ffff::",		                          16,  "39e1::",                                  "39e1::1",                "39e1:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      },
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "ff80::",		                          9,   "3980::",                                  "3980::1",                "39ff:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      },
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "ff00::",		                          8,   "3900::",                                  "3900::1",                "39ff:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      },
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "fc00::",		                          6,   "3800::",                                  "3800::1",                "3bff:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      },
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "c000::",		                          2,   "00::",                                    "00::1",                  "3fff:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      },
+		    std::tuple<std::string, uint8_t, std::string, std::string, std::string, uint64_t>{
+		                                                                                      "::",		                              0,   "::",		                              "::1",                    "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff", 0                      }
+    };
```

Comment:
This part has some redundant tabs and weird formatting. Maybe we can turn clang-format off?

Reply:
fixed all cases in this file.